src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  // ({let partner = person.sex === 'f' ? 'husband' : 'wife'})
+
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried === true
+        ? `${person.partnerName} is my ${person.sex === 'f'
+          ? `husband`
+          : `wife`}`
+        : `I am not married`
```

Comment:
rather hard to read nested ternary.
consider to write in constant nested ternary
```
const partner =  person.sex === 'f' ? 'husband' : 'wife';
```
```suggestion
        ? `${person.partnerName} is my ${partner}`
        : `I am not married`
```